entrypoint.sh

```diff
@@ -5,7 +5,7 @@
 [[ -z "${CEGA_CONNECTION}" ]] && echo 'Environment variable CEGA_CONNECTION is empty' 1>&2 && exit 1
 
 
-cat >> /etc/rabbitmq/rabbitmq.conf <<EOF
+cat >> "${MQDATA}/rabbitmq.conf" <<EOF
```

Comment:
```suggestion
cat >> "/var/lib/rabbitmq/rabbitmq.conf" <<EOF
```
Let's just place the config files where we know we can write data.